README.md

```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
+## Built With
+
+- HTML, CSS and Javascript
+- Git and Github
+
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly include web-pack as this project is built upon it also.